keywords/couchbaseserver.py

```diff
@@ -101,7 +103,7 @@ def delete_internal_rbac_bucket_user(url, bucketname):
         resp.raise_for_status()
     except HTTPError as h:
         log_info("resp code: {}; error: {}".format(resp, h))
-        raise
+        raise RBACUserDeletionError(h)
```

Comment:
A simple string would be fine as a parameter, i.e.

`raise RBACUserCreationError('Could not delete RBAC user!')`